src/tool_operate.c

```diff
@@ -3042,11 +3042,13 @@ static CURLcode transfer_per_config(struct GlobalConfig *global,
      * no environment-specified filename is found then check for CA bundle
      * default filename curl-ca-bundle.crt in the user's PATH.
      *
-     * If Schannel is the selected SSL backend then these locations are
-     * ignored. We allow setting CA location for schannel only when explicitly
-     * specified by the user via CURLOPT_CAINFO / --cacert.
+     * If Schannel or SecureTransport is the selected SSL backend then these
+     * locations are ignored. We allow setting CA location for Schannel and
+     * SecureTransport when explicitly specified by the user via
```

Comment:
nit: We usually refer to it as **Secure Transport** in documentation, using two words

Reply:
Yes, I'm going to fix it in the tidy-up draft PR #14692. (also in url.c where this copy came from)

Reply:
@jay asked the same question when the `url.c` patch was submitted:
https://github.com/curl/curl/pull/12704#issuecomment-1891541873

If we don't mind making this compatibility break, it'd be more consistent to disregard these envvars by default for Secure Transport build. Apple's, Homebrew, and macport's (if I'm reading [it](https://github.com/macports/macports-ports/blob/master/net/curl/Portfile) correctly) curl are not using Secure Transport by default.

Reply:
All in all it doesn't matter given that Secure Transport is going to be deleted next May.